packages/components/buttons/primary-button/src/primary-button.tsx

```diff
@@ -59,9 +59,9 @@ export type TPrimaryButtonProps<
     event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
   ) => void;
   /**
-   * Indicates the size of the icon.
+   * Indicates the size of the button.
    */
-  size?: 'small' | 'big';
+  size?: 'small' | 'medium' | 'big';
```

Comment:
I guess we are deprecating the `small` value so perhaps we need to add a warning to explain about it.
We could use [useWarnDeprecatedProp](https://github.com/commercetools/ui-kit/blob/50ccbfc83c2f01978fa279c2d395bd6967a11c86/packages/utils/src/warn-deprecated-prop.ts#L21) for this.

Reply:
Thanks for the suggestion! 🙇‍♀️ Here's the added warning - [93b4501](https://github.com/commercetools/ui-kit/pull/2443/commits/93b45011dc06a1efcb078fa61823968cb7182eb7).